parsec/core/gui/main_window.py

```diff
@@ -240,6 +244,11 @@ def _show_settings(self):
         d = GreyedDialog(w, title=_("TEXT_SETTINGS_TITLE"), parent=self, width=1000)
         d.exec_()
 
+    def _on_manage_keys(self):
+        w = KeysWidget(config=self.config, parent=self)
+        d = GreyedDialog(w, title=_("TEXT_KEYS_DIALOG"), parent=self, width=1000)
```

Comment:
We should reload the login_widget device list (if any is displayed) to take into account potential changes to the device list.

Reply:
fixed